weusa_server/urls.py

```diff
@@ -13,8 +13,11 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.urls import path
+from django.urls import (
+    path,
+    include
+)
 
 urlpatterns = [
-    
+    path('hospital/', include('hospital.urls')),
```

Comment:
```suggestion
    path('hospital', include('hospital.urls')),
```
메인 url에는 그어떤 슬래시도 붙이지 않고, app 내부 url앞에 슬래시 붙이는 것이 위코드 코딩 컨벤션입니다 :-)

Reply:
감사합니다!!!!!